docs/user_manual/source/control_status_registers.rst

```diff
@@ -1421,19 +1451,42 @@ Reset Value: 0x6800_1000
-  | 31:28 | WARL (0x5,  | **TYPE**. 0x5 (``etrigger``), 0x6 (``mcontrol6``),             |
-  |       | 0x6, 0xF*)  | 0xF (``disabled``).                                            |
+  | 31:28 | WARL (0x2)  | **TYPE**. 2 = Address match trigger (legacy).                  |
   +-------+-------------+----------------------------------------------------------------+
   | 27    | WARL (0x1)  | **DMODE**. Only debug mode can write ``tdata`` registers.      |
   +-------+-------------+----------------------------------------------------------------+
-  | 26:0  | WARL        | **DATA**. Trigger data depending on type                       |
+  | 26:21 | WARL (0x0)  | **MASKMAX**. Hardwired to 0.                                   |
+  +-------+-------------+----------------------------------------------------------------+
+  | 20    | WARL (0x0)  | **HIT**. Hardwired to 0.                                       |
+  +-------+-------------+----------------------------------------------------------------+
+  | 19    | WARL (0x0)  | **SELECT**. Only address matching is supported.                |
+  +-------+-------------+----------------------------------------------------------------+
+  | 18    | WARL (0x0)  | **TIMING**. Break before the instruction at the specified      |
+  |       |             | address.                                                       |
+  +-------+-------------+----------------------------------------------------------------+
+  | 17:16 | WARL (0x0)  | **SIZELO**. Match accesses of any size.                        |
+  +-------+-------------+----------------------------------------------------------------+
+  | 15:12 | WARL (0x1)  | **ACTION**. Enter debug mode on match.                         |
+  +-------+-------------+----------------------------------------------------------------+
+  | 11    | WARL (0x0)  | **CHAIN**. Hardwired to 0.                                     |
+  +-------+-------------+----------------------------------------------------------------+
+  | 10:7  | WARL (0x0*, | **MATCH**. 0: Address matches `tdata2`, 2: Address is greater  |
+  |       | 0x2, 0x3)   | than or equal to `tdata2`, 3: Address is less than `tdata2`.   |
+  +-------+-------------+----------------------------------------------------------------+
+  | 6     | WARL        | **M**. Match in machine mode.                                  |
+  +-------+-------------+----------------------------------------------------------------+
+  | 5     | WARL (0x0)  | Hardwired to 0.                                                |
+  +-------+-------------+----------------------------------------------------------------+
+  | 4     | WARL (0x0)  | **S**. Hardwired to 0.                                         |
+  +-------+-------------+----------------------------------------------------------------+
+  | 3     | WARL (0x0)  | **U**. Hardwired to 0.                                         |
```

Comment:
Change to WARL and add 'Match in user mode.' instead of Hardwired to 0.

Reply:
Updated